client/components/thank-you/index.tsx

```diff
@@ -134,6 +163,22 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 			margin-bottom: 14px;
 		}
 	`;
+	const defaultSupportSectionProps = {
+		title: translate( 'How can we help?' ),
+		description: translate(
+			'Our Happiness Engineers are here if you need help, or if you have any questions.'
+		),
+		links: [
+			{
+				href: CALYPSO_CONTACT,
+				title: translate( 'Ask a question' ),
+			},
+			{
+				href: SUPPORT_ROOT,
+				title: translate( 'Support documentation' ),
```

Comment:
Can you please change this back to `'View support documentation'`?
```suggestion
				title: translate( 'View support documentation' ),
```

Reply:
Oops. I think I let this one pass when rebasing. Sorry! 😅